frontend/src/controllers/EditPromotionController.es6

```diff
@@ -0,0 +1,20 @@
+export default class {
+
+    /* @ngInject */
+    constructor($stateParams, $scope, promotionService, ratePlanService) {
+        this.service = promotionService;
+        $scope.promotions = [];
+        this.$scope = $scope;
+        this.fetchPromotion($stateParams.uuid);
+        ratePlanService.all().then(plans => $scope.ratePlans = plans.data);
+    }
+
+    fetchPromotion(uuid) {
+        let promos = this.service.all().then(res => res.data);
```

Comment:
Can the $html library do this res.data mapping, save having to do it in each controller?